storage/statedb/cache.go

```diff
@@ -29,7 +29,8 @@ type TrieNodeCacheType string
 // TrieNodeCacheConfig contains configuration values of all TrieNodeCache.
 type TrieNodeCacheConfig struct {
 	CacheType                 TrieNodeCacheType
-	NoPrefetch                bool          // Whether to disable heuristic state prefetching for followup blocks
+	NoPrefetch                bool // Whether to disable heuristic state prefetching for followup blocks
+	NumFetcherPrefetchWorker  int
```

Comment:
I think `NumFetcherPrefetchWorker` can be combined with `NoPrefetch`. 
If `NumFetcherPrefetchWorker` == -1, `runtime.NumCPU()/2` goroutines.
if `NumFetcherPrefetchWorker` == 0, no prefetch.
if `NumFetcherPrefetchWorker` > 0, `NumFetcherPrefetchWorker` goroutines.

Reply:
@aidan-kwon Good point. I'll apply your suggestion if this change shows performance improvement, thanks!